components/backup/src/endpoint.rs

```diff
@@ -769,18 +783,24 @@ impl<R: RegionInfoProvider> Progress<R> {
                             break;
                         }
                     }
-                    if info.role == StateRole::Leader {
+                    let peer = find_peer(region, store_id).unwrap().to_owned();
```

Comment:
Will both leader and follower backup the same data in different stores when (replica_read == BackupReplicaRead::LeaderAndFollower)?

Reply:
Indeed, we need the client to make sure that only one replica receives the backup request. Considering TiDB backup currently "push down" backup requsts to all stores, checking on the server side cannot guarantee exclusivity. Will change this.